spec/models/upload_certificate_event_spec.rb

```diff
@@ -19,7 +22,7 @@
 
   context '#value' do
     it 'must be present' do
-      event = UploadCertificateEvent.create
+      event = UploadCertificateEvent.create(usage: '', value: '', component: msa_component)
```

Comment:
I would be nice if you could change these to use the FactoryBot `upload_certificate_event`